Sources/System/FilePath/FilePathComponents.swift

```diff
@@ -72,6 +72,16 @@ extension FilePath {
   }
 }
 
+extension FilePath.Component {
+  public static func == (lhs: Self, rhs: Self) -> Bool {
+    return Self.strSliceEqual(lhs: lhs, rhs: rhs)
+  }
+
+  public func hash(into hasher: inout Hasher) {
```

Comment:
Same problem here -- this is a new symbol, so it needs to either have an `@available` attribute combined with `@backDeployed(before:)`, or it needs to be declared `@_alwaysEmitIntoClient`.